external/selenium/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.stormcrawler</groupId>
+		<artifactId>stormcrawler-external</artifactId>
+		<version>3.4.1-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<artifactId>stormcrawler-selenium</artifactId>
+	<packaging>jar</packaging>
+
+	<name>stormcrawler-selenium</name>
+	<url>https://github.com/apache/stormcrawler/tree/master/external/selenium</url>
+	<description>selenium-based protocol for StormCrawler</description>
+
+	<properties>
+		<selenium.version>4.35.0</selenium.version>
+		<awaitility.version>4.3.0</awaitility.version>
+		<wiremock.version>3.13.1</wiremock.version>
+		<jacoco.haltOnFailure>true</jacoco.haltOnFailure>
+		<jacoco.classRatio>0.00</jacoco.classRatio>
```

Comment:
I know it is annoying when the coverage drops and this verification drops, but you can set the jacoco ratio limits a bit higher.

classRatio: 1.00
There is 1.00 class ratio, because all of the 4 classes are affected. (tbh, I'm not sure if it is right or not, because there is only one single test case, but anyway. This is the actual class ratio.
instructionRatio: 0.69
methodRatio: 0.80
branchRatio: 0.62
lineRatio: 0.66
complexityRatio: 0.55

Or if you want to disable this for some reasons, you only need to set haltOnFailure to false and don't need to override every single ratio.